LWCTimesheetTable/force-app/main/default/lwc/tableLWC/tableLWC.html

```diff
@@ -0,0 +1,58 @@
+<template>
+    <lightning-layout>
+<lightning-tabset >
+<lightning-tab label="Total Timesheets" onactive={handleTotal} >
+    <table class="slds-table slds-table slds-table_cell-buffer slds-table--striped" >  
+        <thead>
+          
+        <template for:each={columns} for:item="col">
```

Comment:
would be good that the name of the **item defined** within the "Total timesheets" tab could be different if you check it against the one defined within the "Recent timesheets" tab.